.github/workflows/ci.yml

```diff
@@ -90,3 +90,17 @@ jobs:
         path: |
           dist/ffmuc-meet.exe
           dist/latest.yml
+
+  winget-release:
+    name: Publish to WinGet
+    runs-on: ubuntu-latest
+    needs: build-windows
+    if: github.event_name == 'release' && github.event.action == 'published'
+    steps:
+    - uses: actions/checkout@v4
+    - name: Publish WinGet Manifest
+      uses: vedantmgoyal9/winget-releaser@v2
+      with:
+        identifier: freifunkMUC.FreifunkMeet
+        max-versions-to-keep: 5 # keep only latest 5 versions
+        token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
is the token set in the repo? i dont know if we maybe can use the App for this also

> token: The GitHub token with which the action will authenticate with GitHub API and create a pull request on the [WinGet Community Repository](https://github.com/microsoft/winget-pkgs). The token should have a public_repo scope.

and maybe we need `fork-user` also or we need to work the WinGet Community Repository first

Reply:
winget-community is forked already. The GITHUB_TOKEN is being used in the workflow as well. So it should be set properly. But I'm open to your suggestion to use our app for that. Would you propose a change for that?

Reply:
when the GITHUB TOKEN is fine i think we should good to go